my_project_omar/modules.py

```diff
@@ -0,0 +1,41 @@
+from pydantic import BaseModel,Field
+from datetime import date
+from uuid import UUID, uuid4
+from typing import Optional
+from enum import Enum
+# Using Enum
+class Status(str, Enum):
+    in_progress = "InProgress"
+    completed = "Completed"
+
+class Priority(str, Enum):
+    high = "High"
+    medium = "Medium"
+    low = "Low"
+class TaskBase(BaseModel):
```

Comment:
Where you used this one? 

I suggest to rename it CreateTaskModel and remove the one in line 22

Reply:
Okay